safe/common/parameters/test/example_group_select.py

```diff
@@ -9,15 +9,19 @@
 
 from PyQt4.QtGui import QApplication, QWidget, QGridLayout
 
-from safe_extras.parameters.qt_widgets.parameter_container import (
-    ParameterContainer)
+from parameters.qt_widgets.parameter_container import ParameterContainer
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->